src/ctap/crypto_wrapper.rs

```diff
@@ -42,6 +44,12 @@ pub const ECDSA_CREDENTIAL_ID_SIZE: usize = 113;
 // See encrypt_key_handle v1 documentation.
 pub const MAX_CREDENTIAL_ID_SIZE: usize = 113;
 
+const ECDSA_CREDENTIAL_ID_VERSION: u8 = 0x01;
```

Comment:
I think @ia0  was suggesting moving this constants into an enum. I'll let him comment on wether he's fine like this (that's an improvement we can easily do in another PR anyways).

Reply:
I think I see the difficulty of using enums. It's that we need a way to convert from the representation to the enum value. And this would need a crate to be done without boilerplate. So I'm fine to keep it like that for now if we don't want to add more dependencies.